src/master-playlist-controller.js

```diff
@@ -872,9 +889,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // Select a new playlist
     const nextPlaylist = this.selectPlaylist();
 
-    videojs.log.warn('Problem encountered with the current playlist.' +
+    if (!error.noWarn) {
```

Comment:
We have to blacklist playlists that are not supported by codec. I don't think this would be a valid warning to log.